### ISSUE TITLE
How to convert the output logits (from VAE decoder) to true SDF values?

### ISSUE THREAD

**Issue by jasongzy** [VAST-AI-Research/TripoSG#46]
Hi, thanks for the great work!

The VAE decoder output appears to be unscaled SDF logits (given the value range). Although this will not prevent using the zero level-set for meshing, could you please clarify the intended way to scale/convert these logits to align with true SDF values? Thanks!

**Comment by Ghuipeng** [comment]
Dose it use the true sdf to Supervise？

**Comment by Ghuipeng** [comment]
In the inference code， it use sdf = 2*sigmod（logits）-1，but i dont konw the truth

**Comment by Ghuipeng** [comment]
After the computation I refer above, use sdf = sdf/100(a num near to it) can get true sdf. I do the experiment
